### PR TITLE
[SPEC-176] 

### PR DESCRIPTION
**Pull request by peterjgrainger**
Configured speckle to report:
- Uncaught exceptions
- Anything logged to winston with severity error
- a sample of every request (for performance stats)

Branch https://github.com/arup-group/global-speckle-server-ansible-playbooks/tree/SPEC-176 to set this config for different environments

### CODE REVIEW

**Comment by peterjgrainger** [comment]
Tested in Staging environment

File: app/telemetry/initSentry.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const { Integrations, init, setUser } = require('@sentry/node')
+const Tracing = require('@sentry/tracing')
+const { Handlers } = require('@sentry/node')
+
+module.exports = ( app ) => {
+  init({ 
+    dsn: "https://16316fb314584d76b3d6f3bedacb7392@o458796.ingest.sentry.io/5497340",
```
Comment (jenessaman) [inline app/telemetry/initSentry.js:7]: Something to specify in .env?

Comment (peterjgrainger) [reply app/telemetry/initSentry.js]: good idea, moved to the ansible scripts

**Comment by peterjgrainger** [comment]
@jenessaman can you take another look, I've created logic to get the user from the jwt in `app/api/middleware/ArupUser.js`

Do you think that is a good idea?

**Comment by jenessaman** [comment]
@peterjgrainger - looks good! Only comment - thoughts on adding a check for if `arupUser` is already added to the request object so you don't have to add it potentially several times?

**Comment by peterjgrainger** [comment]
> @peterjgrainger - looks good! Only comment - thoughts on adding a check for if `arupUser` is already added to the request object so you don't have to add it potentially several times?

@jenessaman makes sense to be defensive. Changed

**Comment by jenessaman** [comment]
Awesome! I think that's it. Excited for Sentry 👍

**Pull request merged**